src/run.sh

```diff
@@ -6,4 +6,4 @@ set -o pipefail
 # Make sure that django's `collectstatic` has been run locally before pushing up to any environment,
 # so that the styles and static assets to show up correctly on any environment.
 
-gunicorn registrar.config.wsgi -t 60
+gunicorn registrar.config.wsgi -t 60 --preload
```

Comment:
**question (non-blocking)**: It wasn't obvious to me from a quick google why we want to use this option. Could we include a comment here on why we use the `--preload` option?

Reply:
I can't remember, tbh. It was either helping me debug something or I needed it to fix something. I've removed `--preload`, here's hoping it was the former and not the latter!